commands/bake.go

```diff
@@ -90,7 +92,7 @@ func runBake(dockerCli command.Cli, targets []string, in bakeOptions) (err error
 		}
 	}()
 
-	dis, err := getInstanceOrDefault(ctx, dockerCli, in.builder, contextPathHash)
+	dis, err := driverutil.GetByNameOrDefault(ctx, dockerCli, in.builder, contextPathHash)
```

Comment:
for at least some of them I don't see why they have moved and are public. They are still called from only one package.